src/rustup.rs

```diff
@@ -84,11 +86,17 @@ pub fn active_toolchain(msg_info: &mut MessageInfo) -> Result<String> {
 }
 
 pub fn installed_toolchains(msg_info: &mut MessageInfo) -> Result<Vec<String>> {
+    static SUFFIX_REGEX: Lazy<Regex> = Lazy::new(|| Regex::new(r" \(.+\)$").expect("suffix"));
+
     let out = rustup_command(msg_info, true)
-        .args(["toolchain", "list", "--quiet"]) // suppress " (active, default)" suffix
+        .args(["toolchain", "list"]) // --quiet would be available from 1.28.0
         .run_and_get_stdout(msg_info)?;
 
-    Ok(out.lines().map(|l| l.trim().to_owned()).collect())
+    // Emulate --quiet by removing suffixes like " (active, default)" or " (override)" manually
+    Ok(out
+        .lines()
+        .map(|l| SUFFIX_REGEX.replace(l.trim(), "").into_owned())
```

Comment:
Regex is not needed for this.

Reply:
How? 🤔

Reply:
Please squash the commits and we can merge. If possible I'd like a test for this also, but no stress we can fo that later

Reply:
I couldn't find any tests for rustup.rs and I know that you don't like new crates like visibility to allow doctests... 
https://github.com/cross-rs/cross/pull/1684